daemon/info_unix.go

```diff
@@ -68,6 +69,80 @@ func (daemon *Daemon) fillPlatformInfo(v *types.Info, sysInfo *sysinfo.SysInfo)
 		logrus.Warnf("failed to retrieve %s version: %s", defaultInitBinary, err)
 		v.InitCommit.ID = "N/A"
 	}
+
+	if !v.MemoryLimit {
+		v.Warnings = append(v.Warnings, "WARNING: No memory limit support")
```

Comment:
Minor nit, are these "WARNING:" prefixes redudnent?

Reply:
@cpuguy83 in general, or this one in particular? also see  https://github.com/moby/moby/pull/37502#issuecomment-413997704

Reply:
Ah, sorry just looked at the code before... It's fine I guess.